File: PinViewSample/PinViewSample.PCL/ViewModels/MainViewModel.cs
using System;
using System.Windows.Input;
using Xamarin.Forms;
namespace PinViewSample.ViewModels
{
    public class MainViewModel
    {
        public ICommand GithubCommand { get; } = new Command(() => 
        { 
            Device.OpenUri(new Uri("https://github.com/lassana/XamarinFormsPinView"));
        });
    }
}

File: FormsPinViewSampleFSharp/FormsPinViewSampleFSharp.Core/ViewModels/MainViewModel.fs
namespace FormsPinViewSampleFSharp.Core.ViewModels

open System
open System.Windows.Input
open Xamarin.Forms

type MainViewModel() = 
    inherit ViewModelBase()

    let githubUri = Uri("https://github.com/lassana/XamarinFormsPinView")

    member this.GithubCommand : ICommand = Command(fun () -> Device.OpenUri githubUri) :> ICommand